app/views/shared/_user_head.html.erb

```diff
@@ -7,5 +7,17 @@
       <h2 class="author"><%= @user.name %></h2>
     </a>
     <span>Number of posts: <%= @user.posts.count %></span>
+
+    
+  <%= form_with url: '/users/sign_out', method: :delete do |f| %>
+    <%= f.submit "Log out", class: 'submit' %>
+  <% end %>
+</div>
```

Comment:
- [ ] Looks like `Log out` is not working or at least after clicking the logout button, then also I'm able to get to that user's page. Please fix this.

Hint: you can use like this to logout the user session.
```
<%= link_to "Log out", destroy_user_session_path, method: :delete %>
```

Reply:
Hi @Dipeshtwis 👋  Thank you for your review. Please would you kindly explain what you mean by you can still get to the user's page? The user's page is not a resource that can only be accessed by authorized users - at least not yet; it's available for all users to check. The signout is working. If you are signed in then you can not visit the sign_in or sign_up page, but you can when you log out.

Reply:
Hi @codepantherr,

Ohh!! That's my bad. Please fix other require changes. 

**Note for next reviewer**: Please do not count this as a require change. 

Thank you

Reply:
Thank you @Dipeshtwis